tests/Utils/CustomOptions.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
Nit: Is the UpperCamelCase filename intended? Asking because most protos in googleapis use lower_snake_case.

Reply:
done, thanks.